ew/utils/loop.py

```diff
@@ -858,6 +859,87 @@ async def enemy_action_tick_loop(id_server):
         except Exception as e:
             ewutils.logMsg('Enemy tick loop failed. Reason:{}'.format(e))
 
+severityLimit = {
+    "low": 5,
+    "medium" : 15,
+    "high" : 25,
+    "wasted" : 50,
+    "wrecked" : 90,
+    "totalled" : 100
+}
+async def drunk_effects(id_server):
+    while not ewutils.TERMINATE:
+        timer = 30
+        await asyncio.sleep(timer)
+        users = bknd_core.execute_sql_query("SELECT inebriation, id_user FROM users WHERE inebriation >0")
```

Comment:
This feature will lag the bot a small amount so if it causes actual issues on market tick we might wanna cut it. But I support it rn.